vent/api/actions.py

```diff
@@ -1121,6 +1121,83 @@ def enable(self,
         self.logger.info("Finished: enable")
         return status
 
+    def tool_status_checker(self, tool_name):
+        """
+        Reads from the plugin manifest. Checks to see if:
+        1. plugin manifest exists
+        2. if the given tool is built
+        3. if the given tool is running
+
+        Args:
+            tool_name(str): tool name. Checks plugin manifest option `name`
+
+        Returns:
+            A tuple of success status, and a tuple containing:
+            bool describing if plugin manifest exists,
+            bool describing if tool is built,
+            bool describing if tool is running.
+            eg: (True, (True, True, False))
+        """
+        status = (True, None)
+        try:
+            # problem is that currently does not throw error if plugin_manifest
```

Comment:
is this still a problem with these changes, or should this comment be read as this try/except is here to throw an error if the plugin_manifest doesn't exist?

Reply:
The ladder. It's checking to make sure that the manifest exists. The comment was me keeping track of why the function wasn't working before.